README.md

```diff
@@ -0,0 +1,37 @@
+# Angular Food Ordering App
+Online Food Order App on React Native using Typescript. It is a hot tutorial series on youtube where you can learn how to make apps like Uber Eats from the sketch. We have covered all the advanced topics to make it production-ready.  
+
+This repository showcases an Expo project that I've been actively developing, utilizing a myriad of technologies. Below is an overview of the directories and files within the project, along with a brief description of their purpose.
+
+## Directory & File Overview
+
+- `.expo-shared`: Contains shared settings for the Expo project.
+- `assets`: Houses assets like images, fonts, and other static files used within the project.
+- `backend`: Contains the backend code and related files.
+- `src`: Source directory for the main application code.
+- `.gitignore`: Specifies files and directories that should be ignored by Git.
+- `App.tsx`: Main entry point for the Expo app.
+- `app.json`: Configuration file for the Expo app settings.
+- `babel.config.js`: Babel configuration file for transpiling the code.
+- `card_icon.png`, `cod_icon.png`, `verify_otp.png`: Images utilized in Part 5 of the project.
+- `package.json`: Lists the project dependencies and scripts.
+- `stripe_server.zip`: Compressed files related to the Stripe server.
+- `tsconfig.json`: Configuration file for TypeScript.
+- `yarn-error.log`: Error log file generated by Yarn.
+- `yarn.lock`: Ensures consistent installations of project dependencies across environments.
+
+## Technologies Used
+
+- **Frontend**: 
+  - Angular 
```

Comment:
Great! you have added the readme file. But I think Angular is not relevant here. Instead of it should be `ReactNative`